src/Tizen.WebView/Tizen.WebView/WebView.cs

```diff
@@ -77,6 +79,18 @@ public class WebView: EvasObject
         /// <since_tizen> 4 </since_tizen>
         public event EventHandler<SmartCallbackArgs> UrlChanged;
 
+        /// <summary>
+        /// Event that occurs when the context menu customized.
```

Comment:
Need more description, 
What means of `The context menu customized`?
and Why event name is ContextMenuCustomize? why not ContextMenuCustomized?

Reply:
This event gives a chance to customize Context menu before showing it.
We will try to find more readable name.
If you have an idea, please feel free to suggest.

Reply:
First, you need to update description
/// Event that occurs when the context menu customized.
It is wrong

And To a way to customize the context menu. 
I recommand to use callback instead of event

``` c#
  public  delegate ContextMenuCustomize(ContextMenu menu);
  public void SetContextMenuCustomizeDelegate(ContextMenuCustomize delegate);

```